components/landing/Landing.vue

```diff
@@ -41,9 +41,20 @@
         </div>
       </div>
       <div v-if="prefix === 'rmrk'">
-        <LazyGalleryLatestSales :passionList="passionList" class="my-5" />
+        <span
```

Comment:
please could you use `<a />` tag instead of `<span>`, no need for `.homepage__toggle` class too

Reply:
done